### PR TITLE
Fixes to show affirmation for all successful submissions

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?
This PR fixes an issue which prevented members from seeing multiple affirmations after multiple successful submissions, since the status of the form wasn't being clear for follow-up checks. This update now allows showing the affirmation modal for each successful text submission!

text submission action multiple submissions

### What are the relevant tickets/cards?
Fixes #thething

### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

**Comment by weerd** [comment]
@mendelB @DFurnes not sure why this error is happening... https://app.wercker.com/dosomething/phoenix-next/runs/build/5ac3cd433ada9c00016a417a?step=5ac3cd79715d1500013339ab

Maybe you 👀 might spot something I messed up?

**Comment by lkpttn** [comment]
@weerd does the modal show on all successful submissions, or only successive ones?

**Comment by weerd** [comment]
> @weerd does the modal show on all successful submissions, or only successive ones?

The bug was with successive ones; so the first would show, but the following wouldn't. Now they all show up!

File: resources/assets/actions/post.js [hunk 1]
```diff
@@ -30,6 +31,19 @@ export function fetchCampaignPosts() {
...
 }
 
+/**
+ * Clear the post submission item with the specified id.
+ *
+ * @param  {String} id
+ * @return {Object}
+ */
+export function clearPostSubmissionItem(id) {
+  return ({
```
Comment (DFurnes) [inline resources/assets/actions/post.js]: Do we need the `()` here? Looks a lil' awk.

File: resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js [hunk 2]
```diff
@@ -20,14 +20,19 @@ class TextSubmissionAction extends React.Component {
     const prevResponse = prevProps.submissions.items[this.props.id] || null;
     const response = this.props.submissions.items[this.props.id] || null;
 
+    // If prior response had no success status, but current response does, then
+    // the submission was successful!
     if (! has(prevResponse, 'status.success') && has(response, 'status.success')) {
       this.resetForm();
+      this.props.openModal('POST_REPORTBACK_MODAL');
```
Comment (DFurnes) [inline resources/assets/components/actions/TextSubmissionAction/TextSubmissionAction.js]: 🔡 ✅

File: resources/assets/actions/post.js (hunk 1 above)
Comment (weerd) [reply resources/assets/actions/post.js]: Oh, because I split into multiple lines, otherwise I can bop them on a single line and remove parentheses...

**Pull request merged**